internal/service/elasticache/user_group_test.go

```diff
@@ -246,6 +264,32 @@ resource "aws_elasticache_user_group" "test" {
 `, rName))
 }
 
+func testAccUserGroupConfig_rotate(rName string) string {
+	return acctest.ConfigCompose(acctest.ConfigAvailableAZsNoOptIn(), fmt.Sprintf(`
+resource "aws_elasticache_user" "test1" {
+  user_id       = "%[1]s-1"
+  user_name     = "default"
+  access_string = "on ~app::* -@all +@read +@hash +@bitmap +@geo -setbit -bitfield -hset -hsetnx -hmset -hincrby -hincrbyfloat -hdel -bitop -geoadd -georadius -georadiusbymember"
+  engine        = "REDIS"
+  passwords     = ["password123456789"]
+}
+
+resource "aws_elasticache_user" "test2" {
```

Comment:
Change user id which will create/delete the `test2` user and trigger the existing bug